### ISSUE TITLE
Unable to create new organisation collections - An unexpected error has occured

### ISSUE THREAD

**Issue by ropelletier** [dani-garcia/vaultwarden#3877]
```
[vaultwarden::api::core::organizations::_][WARN] Data guard `JsonUpcase < NewCollectionData >` failed: Parse("{\"groups\":[],\"name\":\"2.fu454rjes2rzTglIghKsRA==|EodMw6RtH18z1UKOey0SEGCMeKDx8DcddNz/U3IrbHo=|V+YKeJ+1NanGY6dRPLetAEHhQ2tgeQl6PgH15XiCdhE=\"}", Error("missing field `Users`", line: 1, column: 139)).
```

I am having the same issue as #3484. However, I am already running the newest version of vaultwarden on docker. 
Docker version 20.10.21, build baeda1f

* vaultwarden version: 1.29.2

* Install method: Docker

* Clients used: Chrome 116.0.5845.96 (Official Build) (arm64)

* Reverse proxy and version: traefik 2.3.7

### Steps to reproduce
The only collection I have is the Default Collection, anytime I try to create another I get the error above.

**Comment by BlackDex** [comment]
I think you are having some caching issues and that the browser isn't using the latest web-vault version provided by the already built-in web-vault of the container.

It's either that, or you have overwritten the web-vault your self.
The web-vault version should be v2023.7.1

**Comment by ropelletier** [comment]
Indeed. My apologies for wasting your time. I completely forgot that I had the web-vault directory mount in my docker-compose file and did not see it. Thank you for waking me up. Much Appreciated.
